router/router.go

```diff
@@ -616,7 +633,7 @@ func (worker *workerT) handleWorkerDestinationJobs(ctx context.Context) {
 				deliveryLatencyStat.End()
 				// END: request to destination endpoint
 
-				if isSuccessStatus(respStatusCode) {
+				if isSuccessStatus(respStatusCode) && !worker.rt.saveDestinationResponseOverride {
```

Comment:
Do we still need the variable saveDestinationResponse ? If not we can probably remove it and use saveDestinationResponseOverride

Reply:
Let's keep both, for now, saveDestinationResponse is a config from the control plane. We wanted to have some control from the server hence kept it